pyproject.toml

```diff
@@ -99,7 +101,9 @@ tests-cov = "pytest -v -ra --cov --cov-report=xml --cov-report=term --durations=
 
 clean-vendor-compat = "rm -rf vendor_tests/array_api_compat"
 clean-vendor-extra = "rm -rf vendor_tests/array_api_extra"
-copy-vendor-compat = { cmd = "cp -r $(python -c 'import site; print(site.getsitepackages()[0])')/array_api_compat vendor_tests/", depends-on = ["clean-vendor-compat"] }
+copy-vendor-compat = { cmd = "cp -r $(python -c 'import site; print(site.getsitepackages()[0])')/array_api_compat vendor_tests/", depends-on = [
+    "clean-vendor-compat",
+] }
```

Comment:
this might have been from one of my commits due to an IDE extension, but we should keep this on one line